### ISSUE TITLE
WebComponentsReady does not always fire

### ISSUE THREAD

**Issue by bflora2** [webcomponents/webcomponentsjs#384]
My web app listens for the WebComponentsReady event which normally works fine. However, for a small percentage of users in the field, I am receiving automated reports from my app that indicate the WebComponentsReady is not received after waiting 6 to 9 seconds from the time the page has finished loading. The app is idle and waiting on Polymer during that period. Interestingly, this problem is even occurring on Chrome v44, so presumably it's not a polyfill initialization issue. I can gather more data upon request.

**Comment by TimvdLippe** [comment]
Thanks for the issue. It has been a great while since this issue was opened. Sadly, this issue misses a live reproduction with the most recent version of the webcomponents polyfill for us to be able to debug and potentially fix. Versions 1 of the polyfills fix a large amount of issues, probably including this one as well.

Please see [the CONTRIBUTING.md](https://github.com/webcomponents/webcomponentsjs/blob/master/CONTRIBUTING.md) for guidelines on filing issues.  You can start from [this jsBin template](http://jsbin.com/birajez/edit?html,console,output) as a starting point and link it in the issue template under "Live Demo".

Feel free to file a new issue following these instructions if you can still reproduce this issue with versions 1 of the polyfills.  Thank you!
